emission/net/ext_service/habitica/proxy.py

```diff
@@ -26,8 +27,11 @@ def habiticaRegister(username, email, password, our_uuid):
   #want to access their Habitica account from the browser
   #Need to create a way from them to retrieve username/password
   habitica_user_table = edb.get_habitica_db()
-  habitica_user_table.insert({'user_id': our_uuid, 'habitica_username': username, 
-    'habitica_password': password, 'habitica_id': user_dict['data']['_id'], 
+  habitica_user_table.insert({'user_id': our_uuid, 
+    'last_timestamp': arrow.utcnow().timestamp,
```

Comment:
the timestamp from when the habitica user is created. Then, when the autocheck is first called, it calculates the distance from the beginning, and updates the timestamp. After that, the method always calculates the distance walked/biked since the last time it was scored. This avoids users from getting rewarded multiple times for the same thing, and it bypasses the problem of when to reward (every hour? once a day? etc)